File: desafio-logica-e-pensamento-computacional/Javascript/calculoSalario.js
module.exports = function calcularSalario(salario, beneficio) {
  let salarioDescontado = 0;

  if (salario > 0 && salario <= 1100) {
    salarioDescontado = salario * 0.95;
  } else if (salario >= 1100.01 && salario <= 2500) {
    salarioDescontado = salario * 0.9;
  } else if (salario >= 1100.01 && salario <= 2500) {
    salarioDescontado = salario * 0.85;
  } else {
    console.log("O salário informado é inválido");
  }

  const salarioCalculado = salarioDescontado + beneficio
  console.log("O salário calculado é R$ " + salarioCalculado)
}

File: desafio-logica-e-pensamento-computacional/Python/CalculoSalario.py
def calcular_imposto(salario, beneficio):
    salarioDescontado = 0

    if (salario > 0 and salario <= 1100):
        salarioDescontado = salario * 0.95
    elif (salario >= 1100.01 and salario <= 2500):
        salarioDescontado = salario * 0.9
    elif (salario >= 1100.01 and salario <= 2500):
        salarioDescontado = salario * 0.85
    else:
        print("O salário informado é inválido")

    salarioCalculado = salarioDescontado + beneficio
    print("O salário calculado é R$ " + str(salarioCalculado))

print("Digite o salário:")
valor_salario = float(input())

print("Digite o benefício:")
valor_beneficio = float(input())

calcular_imposto(valor_salario, valor_beneficio)